vault/dendron.tutorial.user-interface.md

```diff
@@ -0,0 +1,64 @@
+---
+id: gHdxXlNMr1w4xqee0n-Mb
+title: User Interface
+desc: 1. User Interface
+updated: 1639427631131
+created: 1625563862198
+nav_order: 0
+---
+
+Let's do a brief overview on how to navigate the Dendron user interace (UI), which is called the **[[Workbench|dendron.topic.workbench]]**.
+
+_Quick note on running commands in VSCode:_
+
+> If you're unfamiliar with VSCode, the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) is used to quickly run commands, including Dendron commands. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. If you see instructions telling you to run a command, remember to bring up the command palette and type the command name to run it.
+
+![[dendron://dendron.dendron-site/asset.preview#tutorial-dendron-layout,1:#*]]
```

Comment:
we need to update the GIF here since its not Dark+ theme and has a yellow outline